### DISCUSSION TITLE
Remove the types of "newsletter" and "blog" from the medium tag.

### DISCUSSION THREAD

**Discussion by Dwev** [Podcastindex-org/podcast-namespace#608] [Deprecation Proposal]
Yes, the `medium` tag is a recent addition, but all the other types have a few feeds using them.  Blog and Newsletter have zero (based on podcastindex API results), and I don't see it changing soon.

My understanding is that a "podcast" is loosely defined as an **RSS feed with a media enclosure**.  By that token, `newsletters` and `blogs` are not really that on a feed level.  I can see you might have some episodes that contain media, but rarely the whole feed.

I can't find discussion around the inclusion of these into the podcast namespace (and that could be due to my own lack of search skill), but I feel these medium types add noise and don't add value.

I am more than happy to be shown the error of my ways though, which I what I guess these discussion threads are for!

**Comment by agates** [comment]
1. Podcast is a different medium.  Not all RSS feeds are podcasts.  This is why the medium exists.
2. We can and should put things such as markdown files in the enclosure.

**Reply by Dwev** [reply]
1. No arguments here.
2. I think "should" is where I miss something. 

I don't understand why the podcast namespace should be used for syndicating Markdown or other "article" type feeds that are already served by RSS itself.

Do you have any insight into the use case for a markdown file aggregator?

**Reply by daveajones** [reply]
Podcasts have always had various media enclosures in them going way back to the beginning.  Things like PDF's and such.  It's not used very much, but it's definitely still a "podcast" in the technical sense of being media served over RSS.  The confusion is purely syntax, since "podcast" and podcast aren't identical.

**Reply by Dwev** [reply]
Hmm, okay. I could see this for an online magazine or comic book series distributed in PDF for example, then being picked up by a magazine reader/comic book reader.

Neither of those are "blog" or "newsletter" though, and could justify an additional medium type.

If there are still no feeds of this medium type in 2-3 years, would it be worth depreciating then?

**Reply by Dwev** [reply]
Just heard the latest PC2.0 episode - great to hear some thoughtful discussion about this. Helps me get some context around the idea of these tags, and you've converted me too, @daveajones 

That said, Adam mentioned changing the medium type from `blog` to `podcast`, but medium applies to the whole feed right?

**Comment by samsethi** [comment]
I would prefer we kept these two mediums.  TrueFans is not just a listening app, we are developing a platform.  We will allow Publishers to write a blog if they choose and have fans subscribe to their newsletter.  We have created a blog that supports zaps/likes and boost comments.  All of this is RSS and I see no reason why we can't have the RSS for a podcaster be extended to include blog posts. 

<img width="1388" alt="TrueFans___Publishers___Wondery" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/82abf6aa-ff08-49a8-9738-0cd562eb42fe">

**Comment by Dwev** [comment]
Wow, @samsethi! That's really interesting to utilise V4V for blog posts. That's something I never considered, and it makes a lot of sense!
So, there are quite a few compelling reasons to leave these mediums as part of the spec. :)
Since I opened the discussion, and I've since changed my mind, should I close it now or leave it open for more comment? What's the recommended action?
